lib/riemann/tools/riemann_client_wrapper.rb

```diff
@@ -25,7 +25,7 @@ def initialize(options)
             events << @queue.pop while !@queue.empty? && events.size < @max_bulk_size
 
             client.bulk_send(events)
-          rescue Riemann::Client::Error => e
+          rescue Riemann::Client::Error, OpenSSL::SSL::SSLError, SocketError => e
```

Comment:
Meh, i also see a `EOFError`.  Maybe ignoring all StandardError makes more sense after all.

Reply:
Adjusted to catch all `StandardError`, this should be more resilient.